sklearn/utils/testing.py

```diff
@@ -773,3 +773,121 @@ def __init__(self, check, arg_text):
 
     def __call__(self, *args, **kwargs):
         return self.check(*args, **kwargs)
+
+# Utils to test docstrings
+
+def _get_args(function, varargs=False):
+    """Helper to get function arguments"""
+    # NOTE this works only in python3.5
+    if sys.version_info < (3, 5):
```

Comment:
I am pretty sure it works too on 2.7 in mne:

https://github.com/mne-tools/mne-python/blob/master/mne/tests/test_docstring_parameters.py